MaxText/layers/models.py

```diff
@@ -347,6 +347,10 @@ def get_decoder_layers(self):
       from MaxText.layers import gpt3  # pylint: disable=import-outside-toplevel
 
       return [gpt3.Gpt3DecoderLayer]
+    elif self.config.decoder_block == DecoderBlockType.QWEN3:
+      from MaxText.layers import qwen3  # pylint: disable=import-outside-toplevel
```

Comment:
Param and I changed the convention to include the layer at the module level recently :)